setup.py

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env python
+import os
 from setuptools import setup, find_packages
 from os import environ as env
 
+requirements_file = os.path.join(os.path.dirname(__file__), 'requirements.txt')
+with open(requirements_file) as f:
+    requirements = [line.rstrip('\n')
+                    for line in f
+                    if line and not line.startswith('#')]
```

Comment:
Have you considered using `pkg_resources.parse_requirements`?

e.g. https://github.com/Affirm/datatools/blob/master/dbimport/setup.py#L27